scripts/add_routes_to_push_manifest.js

```diff
@@ -0,0 +1,61 @@
+/**
```

Comment:
Should the `polymer-build` do this at some point?

Reply:
It should, yeah. I think we're still discussing what the syntax should look like

Reply:
Thanks Monica!